Stratis.Bitcoin/RPC/Controllers/FullNodeController.cs

```diff
@@ -1,51 +1,132 @@
 ﻿using Microsoft.AspNetCore.Mvc;
-using NBitcoin;
+using Microsoft.Extensions.PlatformAbstractions;
+using Stratis.Bitcoin.Configuration;
 using Stratis.Bitcoin.RPC.Models;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Stratis.Bitcoin.BlockStore;
+using Stratis.Bitcoin.MemoryPool;
+using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.Utilities;
 
 namespace Stratis.Bitcoin.RPC.Controllers
 {
     //TODO: Need to be extensible, should be FullNodeController
-    public partial class ConsensusController : Controller
+    public partial class FullNodeController : Controller
     {
+        private Builder.IFullNode _FullNode;
+        private NodeSettings _Settings;
+        private NBitcoin.Network _Network;
+        private ConsensusValidator _ConsensusValidator;
+        private NBitcoin.ChainBase _Chain;
+        private ChainBehavior.ChainState _ChainState;
+        private BlockStoreManager _BlockManager;
+        private MempoolManager _MempoolManager;
+        private Connection.ConnectionManager _ConnectionManager;
+
+        public FullNodeController(
```

Comment:
I think this can be built so that no services are required, but perhaps particular actions may be disabled accouring to available services. I see this as becoming a single RPCController for all the actions, and for example a "help" command might filter on what services are available.

I based the response model omitable properties on which properties Core is guaranteed to return. If there's no concern about returning a sparser model than this, I will remove all the properties in the result which are not backed by an available service.